DBM-Core/DBM-Nameplate.lua

```diff
@@ -272,9 +272,14 @@ end
 -----------------------------------------
 -- handling both: Icon CDs and static nameplate icons
 --Add more nameplate mods as they gain support
-local function SupportedNPMod()
+local function SupportedNPModIcons()
 	if not DBM.Options.UseNameplateHandoff then return false end
-	if _G["KuiNameplates"] or _G["TidyPlatesThreatDBM"] or _G["Plater"] then return true end
+	if _G["KuiNameplates"] or _G["TidyPlatesThreatDBM"] or (_G["Plater"] and Plater.db.profile.bossmod_support_enabled) then return true end
+	return false
+end
+local function SupportedNPModBars()
+	if not DBM.Options.UseNameplateHandoff then return false end
+	if _G["KuiNameplates"] or _G["TidyPlatesThreatDBM"] or (_G["Plater"] and Plater.db.profile.bossmod_support_bars_enabled) then return true end
```

Comment:
There is no need for Plater.db.profile.bossmod_support_bars_enabled check here. 
Unless the user has disabled handoff, if plater is installed it should use platers preference. I already added a login print that advices user to enable it in plater if it's off, or disable it in DBM. This current code would just disrespect platers setting and force plater users to use DBMs internal even though handoff is enabled. That's not good

Also only plater supports bars so shouldn't check kui or threatplates at all.

Reply:
Makes sense, thanks for the clarification.